include/SFML/Graphics/Transform.hpp

```diff
@@ -31,16 +31,17 @@
 #include <SFML/Graphics/Export.hpp>
 #include <SFML/Graphics/Rect.hpp>
 #include <SFML/System/Vector2.hpp>
-#include <SFML/System/Angle.hpp>
 
 
 namespace sf
 {
+class Angle;
+
```

Comment:
Is this trying to minimize compile time? Is that compile time improvement measurable or meaningful?